### ISSUE TITLE
[A11y] Contrast issues

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#1451]
## Description
Contrast issues across all UUI components.

### Precondition
Screen Reader is **OFF**

### Steps to Reproduce
1. Open [Accordion](https://uui.epam.com/documents?id=accordion&mode=doc&skin=UUI4_promo&category=components) component.
2. Expand "Accordion with react components children" accordion.
3. Measure the color contrast between [Accept] button text and its background.

### Actual Result
The contrast ratio between [Accept] button text and its background is less than 4.5:1: white text #FFFFFF over green background #65A300 gives 3.1:1.

![2023-06-29_17h59_28](https://github.com/epam/UUI/assets/57751283/5df1ec01-fe41-400e-8f59-3cffa2392f33)

### Expected Result 
Recommended contrast ratio between colors is at least 4.5:1. 
Failed WCAG Checkpoint: 1.4.3 Contrast (Minimum) (Level AA)
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

**Comment by DzmitryTrubchyk** [comment]
Walk through cases, when AA level missed
